index.html

```diff
@@ -10,300 +11,294 @@
+                                    <img src="Images/M-Vector.svg" alt="medium">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="#" class="github">
+                                    <img src="Images/C-Vector.svg" alt="github">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="#" class="angelist"><img src="Images/P-Vector.svg" alt="angelist">
+                                </a>
+                            </li>
+                        </ul>
+                        <button type="submit">Get my resume</button>
+                    </div>
+                </div>
+                <div class="skill-list">
+                    <ul class="frames">
+                        <li class="languages">Languages</li>
+                        <li class="arrow-down"><img src="Images/arrow-down-icon.png" alt="Arrow-Down Icon">
                         </li>
-                        <li>
-                            <a href="#" class="github">
-                                <img src="Images/C-Vector.svg" alt="github">
-                            </a>
+                    </ul>
+                    <div class="lang">
+                        <ul class="content">
+                            <li><img class="icon" src="Images/Ellipse-1.png" alt="JavaSript-logo"></li>
+                            <li><span>JavaScript</span></li>
+                        </ul>
+                        <ul class="content">
+                            <li><img class="icon" src="images/HTML-logo.png" alt="html-logo"></li>
+                            <li><span>HTML</span></li>
+                        </ul>
+                        <ul class="content">
+                            <li><img class="icon" src="Images/CSS-logo.png" alt="css-logo"></li>
+                            <li><span>CSS</span></li>
+                        </ul>
+                    </div>
+                    <hr>
+                    <ul class="frames">
+                        <li class="languages">Frameworks</li>
+                        <li class="arrow-down"><img src="Images/arrow-right.png" alt="Arrow Right Icon">
                         </li>
-                        <li>
-                            <a href="#" class="angelist"
-                            ><img src="Images/P-Vector.svg" alt="angelist">
-                        </a>
+                    </ul>
+                    <hr />
+                    <ul class="frames">
+                        <li class="languages">Skills</li>
+                        <li class="arrow-down"><img src="Images/arrow-right.png" alt="Arrow Right Icon">
                         </li>
                     </ul>
-                    <button>Get my resume</button>
+                    <hr>
                 </div>
```

Comment:
- [ ] Kindly alter the headings in the skills list so they look similar to the Figma template
> Figma Design
![image](https://user-images.githubusercontent.com/90556221/178115709-993ea683-15fc-4d14-a988-4cad6a694f0e.png)


> Your Design
![image](https://user-images.githubusercontent.com/90556221/178115687-b0b1091e-179a-4430-87e6-b685435ea6e4.png)

- [ ] One of the images in the skills section does not seem to be loading properly, kindly alter the path so the image loads properly.

- [ ] The background color around the top-right edge of the about section does not correspond to the colors in the Figma template, kindly resolve this issue.